emissionsapi/openapi.yml

```diff
@@ -143,6 +146,19 @@ components:
         The parameter must be in the form `lon1,lat1,lon2,lat2`,
         representing the upper left and lower right corners of a rectangle.
       example: [15, 45, 20, 40]
+    point:
+      in: query
+      name: point
+      schema:
+        type: array
+        items:
+          type: number
+        maxItems: 2
+        minItems: 2
+      description: |
+          Let you define a single `point` in the form longitude,latitude.
+          Only data in a small area around this point is shown.
```

Comment:
It was intentionally vague, since I am not sure how we should deal with this in the future. Do you have a better idea for a description? Or do you prefer being more precise about it?